### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by irina-mandarina**

### CODE REVIEW

File: homework/12_B/13_Irina/Circles/src/OverlayType.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public enum OverlayType {
```
Comment (svetlin-mladenov) [inline homework/12_B/13_Irina/Circles/src/OverlayType.java:1]: IMHO `OverlapType` would have been a better name.

File: homework/12_B/13_Irina/Circles/src/Main.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        if (c1.getR() + c2.getR() == distanceBtwCenters) {
+            return OverlayType.TOUCHING;
+        }
+
+        if ( (c1.getR() > c2.getR() && (distanceBtwCenters + c2.getR() <= c1.getR())) ||
+                c2.getR() > c1.getR() && (distanceBtwCenters + c1.getR() <= c2.getR())) {
+            return OverlayType.ONE_INSIDE_OTHER;
+        }
+
+        if (c1.getR() + c2.getR() > distanceBtwCenters) {
```
Comment (svetlin-mladenov) [inline homework/12_B/13_Irina/Circles/src/Main.java:25]: What if they are one inside the other but not touching.

**Comment by svetlin-mladenov** [comment]
You don't need to commit the `.class` and `.iml` files. Class files are compiled java files. The JVM will recompile automatically if these are missing.

File: homework/12_B/13_Irina/Circles/src/Main.java (hunk 2 above)
Comment (irina-mandarina) [reply homework/12_B/13_Irina/Circles/src/Main.java:25]: i don't understand what you mean. could you provide a graph?

**Pull request merged**